src/HubCollection.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Mercure Component project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Mercure;
+
+use Symfony\Component\Mercure\Exception\InvalidArgumentException;
+
+final class HubCollection
+{
+    private $hubs;
+    private $defaultHub;
+
+    /**
+     * @param HubInterface[] $hubs
+     */
+    public function __construct(string $defaultHub, iterable $hubs)
```

Comment:
Having a PSR-11 hub locator would be more efficient given how we are accessing hubs (by name).

Reply:
api-platform must not make use of symfony framework features ( only components ), so we cannot relay on the name give in the configuration, or symfony service locator.

what do you propose instead? 🤔

Reply:
I suggest a `Psr\Container\ContainerInterface $hubLocator`. How it is wired does not matter, no need for coupling with Symfony DI as it can be any psr-11 container

Reply:
Using the PSR-11 container is ok for API Platform.

Reply:
Okay 👍

Reply:
done